packages/main/test/pages/styles/Popover.css

```diff
@@ -66,3 +66,10 @@ ui5-date-picker,
     height: 150px;
     width: 300px;
 }
+
+.containingBlock {
+	border: 1px solid black;
+	container-type: inline-size;
+	position: relative;
+	z-index: 1;
```

Comment:
Can you add more styles here to improve the example and show a visual difference in case this fix breaks?
If I remove these styles, the Popover gets displayed in the exact same way as if the styles are there.

Reply:
If you remove only - "position: relative;	z-index: 1;", the popover will be rendered below the text of the buttons below.